### DISCUSSION TITLE
Have I been hacked ?

### DISCUSSION THREAD

**Discussion by aminomaboul** [immich-app/immich#8425] [Q&A]
days ago, i exposed my Immich server to the internet via NPM (Nginx Proxy Manager) on port 443 using TLS encryption, and today when opening both the Android app and the Web app, i'm suprised to see the Immich logo changed 

![image](https://github.com/immich-app/immich/assets/25910126/3c52c3b0-1377-4f95-a0af-b1df871ee50c)

i'm the only one to have access to my LAN ! 
is anyone aware of that ?

**Comment by coltenkrauter** [comment]
https://github.com/immich-app/immich/discussions/8430

**Reply by bhchew** [reply]
About to link the same : )
